config/crd/bases/app.k8s.io_applications.yaml

```diff
@@ -1,12 +1,12 @@
-# Copyright 2020 The Kubernetes Authors.
+# Copyright 2021 The Kubernetes Authors.
 # SPDX-License-Identifier: Apache-2.0
 
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/application/pull/2
```

Comment:
field additions to k8.up API need to get API review, and should probably update this to point to this PR. cc @thockin